include/networkit/centrality/GedWalk.hpp

```diff
@@ -0,0 +1,231 @@
+/* GedWalk.hpp
+ *
+ *  Created on: 18.6.2018
+ *     Authors: Eugenio Angriman <angrimae@hu-berlin.de>
+ *              Alexander van der Grinten <avdgrinten@hu-berlin.de>
+ */
+
+// networkit-format
+
+#ifndef NETWORKIT_CENTRALITY_GED_WALK_HPP_
+#define NETWORKIT_CENTRALITY_GED_WALK_HPP_
+
+#include <networkit/algebraic/CSRMatrix.hpp>
+#include <networkit/base/Algorithm.hpp>
+#include <networkit/graph/Graph.hpp>
+
+#include <tlx/container/d_ary_addressable_int_heap.hpp>
+
+namespace NetworKit {
+
+class GedWalk final : public Algorithm {
+
+    using walks = double;
+    using uchar = unsigned char;
```

Comment:
Non-idiomatic typedef. Please remove this.

Reply:
I removed it.